samples/v2/two_step_pipeline_containerized.py

```diff
@@ -28,23 +27,34 @@ def component1(text: str, output_gcs: Output[Dataset]):
     return ContainerSpec(
         image='google/cloud-sdk:slim',
         command=[
-            'sh -c | set -e -x', 'echo', text, '| gsutil cp -', output_gcs.uri
+            'sh',
+            '-c',
+            'echo "$0" | gsutil cp - "$1"',
+        ],
+        args=[
+            text,
+            output_gcs.uri,
         ])
 
 
 @container_component
 def component2(input_gcs: Input[Dataset]):
     return ContainerSpec(
         image='google/cloud-sdk:slim',
-        command=['sh', '-c', '|', 'set -e -x gsutil cat'],
-        args=[input_gcs.path])
+        command=[
+            'sh',
+            '-c',
+            'gsutil cat "$0"',
+        ],
+        args=[
+            input_gcs.uri,
+        ])
```

Comment:
See comment above. This could be:

```python
    dsl.ContainerSpec(
        image='alpine',
        command=['cat', '"$1"'],
        args=[input_gcs.path])
```